Disable USE_PREBOOT as a workaround to boot failure triggered by
initializing USB. (Closes: #973323, #980434)

Reported upstream:

  https://lists.denx.de/pipermail/u-boot/2021-January/438098.html

Index: u-boot-2021.07+dfsg/configs/rockpro64-rk3399_defconfig
===================================================================
--- u-boot-2021.07+dfsg.orig/configs/rockpro64-rk3399_defconfig
+++ u-boot-2021.07+dfsg/configs/rockpro64-rk3399_defconfig
@@ -12,7 +12,7 @@ CONFIG_DEBUG_UART_CLOCK=24000000
 CONFIG_SPL_SPI_FLASH_SUPPORT=y
 CONFIG_SPL_SPI_SUPPORT=y
 CONFIG_DEBUG_UART=y
-CONFIG_USE_PREBOOT=y
+CONFIG_USE_PREBOOT=n
 CONFIG_DEFAULT_FDT_FILE="rockchip/rk3399-rockpro64.dtb"
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_MISC_INIT_R=y
Index: u-boot-2021.07+dfsg/configs/pinebook-pro-rk3399_defconfig
===================================================================
--- u-boot-2021.07+dfsg.orig/configs/pinebook-pro-rk3399_defconfig
+++ u-boot-2021.07+dfsg/configs/pinebook-pro-rk3399_defconfig
@@ -12,7 +12,7 @@ CONFIG_SPL_SPI_FLASH_SUPPORT=y
 CONFIG_SPL_SPI_SUPPORT=y
 CONFIG_DEBUG_UART=y
 CONFIG_BOOTDELAY=3
-CONFIG_USE_PREBOOT=y
+CONFIG_USE_PREBOOT=n
 CONFIG_DEFAULT_FDT_FILE="rockchip/rk3399-pinebook-pro.dtb"
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_MISC_INIT_R=y
